src/domUpdates.js

```diff
@@ -144,18 +144,26 @@ let domUpdates = {
 
   //Pantry Display
   displayPantryInfo(pantry) {
-    pantry.forEach(ingredient => {
-      let ingredientHtml = `<li><input type="checkbox" class="pantry-checkbox" id="${ingredient.name}">
-        <label for="${ingredient.name}">${ingredient.name}, ${ingredient.count}</label></li>`;
-      document.querySelector(".pantry-list").insertAdjacentHTML("beforeend",
-        ingredientHtml);
-    });
-  },
+      let pantryList = document.querySelector(".pantry-list")
+      pantryList.innerHTML = ''
```

Comment:
This seems to be where our issues with the pantry list populating twice came from. Clearing out the html looks like it does the trick!